test/plugins/DARTDoublePendulum.cc

```diff
@@ -18,7 +18,15 @@
 #include <memory>
 #include <unordered_map>
 
+// Suppressing a -Wreturn-type warning in octomap
+// https://github.com/OctoMap/octomap/issues/431
+#include <gz/utils/SuppressWarning.hh>
+#ifndef _MSC_VER
+DETAIL_GZ_UTILS_BEGIN_WARNING_SUPPRESSION("-Wreturn-type")
 #include <dart/dart.hpp>
+DETAIL_GZ_UTILS_WARN_RESUME
+#endif  // _MSC_VER
```

Comment:
I was also looking into this. For this particular case, I think we can just include only the headers that are necessary?
https://github.com/gazebosim/gz-physics/compare/homebrew_octomap_warning

Reply:
ok, I'll patch that in for this class, but I think it won't work for WorldFeatures.cc

Reply:
https://github.com/gazebosim/gz-physics/pull/728/commits/65e36cd5674b5634878d59ae0b18c003efb6d4db

Reply:
thanks, yeah meant only for this particular case in `test/plugins/DARTDoublePendulum.cc`